### DISCUSSION TITLE
Support Spring Boot 3.2

### DISCUSSION THREAD

**Discussion by Canos** [awspring/spring-cloud-aws#976] [General]
Hi folks, is there an estimated time to support Spring Boot 3.2?
Thanks for you effort with this amazing tool

**Comment by maciejwalkowiak** [comment]
Spring Cloud - the umbrella project - is going to be released today. Once it's done we should release this week - I'll try ASAP but things happen so can't promise tomorrow.

**Reply by pcornelissen** [reply]
Just saw the release, thanks! Is there a kind of changelog/release notes somewhere?

**Reply by maciejwalkowiak** [reply]
3.1.0 brings all changes in 3.0.4 + Spring Boot 3.2 compatibility, + potentially breaking change in Parameter Store:

https://github.com/awspring/spring-cloud-aws/releases/tag/v3.1.0
https://github.com/awspring/spring-cloud-aws/releases/tag/v3.0.4
